grakn-graql-shell/pom.xml

```diff
@@ -32,7 +32,7 @@
     <dependencies>
         <dependency>
             <groupId>ai.grakn</groupId>
-            <artifactId>grakn-core</artifactId>
+            <artifactId>grakn-graql</artifactId>
```

Comment:
Because now autocomplete happens on the client side, so it needs to know about Graql keywords using the parser etc.